khelo/lib/ui/flow/team/team_list_screen.dart

```diff
@@ -110,111 +116,101 @@ class _TeamListScreenState extends ConsumerState<TeamListScreen>
 
     return ListView.separated(
       itemCount: state.filteredTeams.length,
-      padding: context.mediaQueryPadding +
-          const EdgeInsets.only(left: 16, right: 16, top: 16, bottom: 60),
-      separatorBuilder: (context, index) => const SizedBox(height: 16),
+      padding: const EdgeInsets.all(16) + context.mediaQueryPadding,
+      separatorBuilder: (context, index) =>
+          Divider(color: context.colorScheme.outline),
       itemBuilder: (context, index) {
         final team = state.filteredTeams[index];
-        return Column(
-          crossAxisAlignment: CrossAxisAlignment.start,
-          children: [
-            if (index == 0) ...[
-              Text(
-                state.selectedFilter.getString(context),
-                style: AppTextStyle.subtitle1.copyWith(
-                    color: context.colorScheme.textPrimary, fontSize: 20),
-              ),
-              const SizedBox(
-                height: 16,
-              )
-            ],
-            _teamListCell(
-              context,
-              notifier,
-              team,
-              showMoreOptionButton: state.currentUserId == team.created_by,
-            ),
-          ],
+        return _teamListCell(
+          context,
+          team: team,
+          showMoreOptionButton: state.currentUserId == team.created_by,
         );
       },
     );
   }
 
   Widget _teamListCell(
-    BuildContext context,
-    TeamListViewNotifier notifier,
-    TeamModel team, {
+    BuildContext context, {
+    required TeamModel team,
     required bool showMoreOptionButton,
   }) {
     return OnTapScale(
       onTap: () {
         AppRoute.teamDetail(teamId: team.id ?? "INVALID ID").push(context);
       },
-      child: Row(
-        children: [
-          ImageAvatar(
-            initial: team.name[0].toUpperCase(),
-            imageUrl: team.profile_img_url,
-            size: 50,
-          ),
-          const SizedBox(
-            width: 8,
-          ),
-          Expanded(
-            child: Column(
-              crossAxisAlignment: CrossAxisAlignment.start,
-              children: [
-                Text(
-                  team.name,
-                  style: AppTextStyle.header4
-                      .copyWith(color: context.colorScheme.textPrimary),
-                ),
-                Text(team.city ?? context.l10n.common_not_specified_title,
+      child: ListTile(
```

Comment:
I guess, this is resulting to `No material widget found error` in iOS Simulator.

![NoMaterialWidgetInIOS](https://github.com/canopas/khelo/assets/122426509/8593e42a-662d-4f95-8ea5-2d2e82563453)